### PR TITLE
feat: add recipe for metanetx-post

### PR DESCRIPTION
**Pull request by Midnighter**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/metanetx-post/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - depinfo
+    - httpx
+    - metanetx-sdk
+    - pandas
+    - pydantic
+    - pyparsing
+    - python >=3.8
+    - rdflib
+    - rdkit
+    - SQLAlchemy
```
Comment (synapticarbors) [inline recipes/metanetx-post/meta.yaml]: I think this should be `sqlalchemy`

**Comment by Midnighter** [comment]
> This looks generaly good to me. My only question was regarding the versions specified in the upstream `setup.cfg`, which pin every dependency's version using the "~=". I'm not sure if you want to replicate this here.

Thank you for the thought. I previously had a lot of problems with pinning dependencies in recipes. Also, there is the global list of version pins so I don't think it's needed.

File: recipes/metanetx-post/meta.yaml (hunk 1 above)
Comment (Midnighter) [reply recipes/metanetx-post/meta.yaml]: The name on PyPI is that https://pypi.org/project/SQLAlchemy/ but they should anyway be normalized so I don't think it's an issue.

**Comment by Midnighter** [comment]
@conda-forge/help-python as far as I'm concerned this is ready for merge.

File: recipes/metanetx-post/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/metanetx-post/meta.yaml]: As a rule we normalize them in the recipe as well.

File: recipes/metanetx-post/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed"
+  entry_points:
+    - "mnx-post = metanetx_post.cli.main:cli"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools
```
Comment (scopatz) [inline recipes/metanetx-post/meta.yaml]: Please remove setuptools as pip is already a dependency

File: recipes/metanetx-post/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  run:
+    - aioftp
+    - Click
```
Comment (scopatz) [inline recipes/metanetx-post/meta.yaml]: please lowercase all dependencies

File: recipes/metanetx-post/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    - metanetx_post
+  commands:
+    - "mnx-post --help"
+
+about:
+  home: https://pypi.org/project/metanetx-post/
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: "Enrich the information coming from [MetaNetX](https://metanetx.org) from additional sources."
```
Comment (scopatz) [inline recipes/metanetx-post/meta.yaml]: Please remove markdown from this field

File: recipes/metanetx-post/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  description: |
+    This package provides a command line program `mnx-post` that has a number of
+    commands to augment the existing information from MetaNetX from other
+    sources, among them BiGG, ExPASy, KEGG, and SEED.
```
Comment (scopatz) [inline recipes/metanetx-post/meta.yaml]: And also remove the markdown here

**Comment by Midnighter** [comment]
Thank you for the review! I made all suggested changes. I was just wondering about the markdown. I have used it in other recipes and when it is added to the feedstock README it renders fine. See, for example, [this one](https://github.com/conda-forge/metanetx-sdk-feedstock). Is not having markdown a general rule?

**Comment by scopatz** [comment]
yeah you should not have markdown, that is a general rule because it is not guaranteed to be rendered

**Pull request merged**